clients/python/Gemfile

```diff
@@ -0,0 +1,2 @@
+source 'https://rubygems.org'
+gem 'github-pages', group: :jekyll_plugins
```

Comment:
Please add Gemfile, Gemfile.lock  and _config.yml to `.gitattributes` with `linguist-generated=false`.
This way they will appear normally in the diff.
While you're at it, also  lakefs_client/client.py :)